raspberry-pi/3/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, pkgs, ...}:
+
+{
+  boot = {
+    consoleLogLevel = lib.mkDefault 7;
+    extraTTYs = [ "ttyAMA0" ];
+    kernelPackages = lib.mkDefault pkgs.linuxPackages_rpi;
+    kernelParams = [
+      "dwc_otg.lpm_enable=0"
+      "console=ttyAMA0,115200"
+      "rootwait"
+      "elevator=deadline"
+      "cma=32M"
+    ];
+    loader = {
+      grub.enable = lib.mkDefault false;
+      generationsDir.enable = lib.mkDefault false;
+      raspberryPi = {
+        enable = lib.mkDefault true;
+        version = lib.mkDefault 3;
+      };
+    };
+  };
+
+  nix.buildCores = 4;
```

Comment:
While the device has 4 cores it has less than a 1GB of memory and will run out of memory like this.